### PR TITLE
add the controller command

### PR DESCRIPTION
**Pull request by deads2k**
Add the command for the SSCS controller and updates the operator to make it work. 

WIP in while I sort out the deps.

```
oc cluster up --kube-only
oc auth reconcile -f install/serving-cert-signer/install-rbac.yaml
oc create -f manifests/v3.10.0/service-serving-cert-signer-controller/clusterrole.yaml
oc create -f manifests/v3.10.0/service-serving-cert-signer-controller/clusterrolebinding.yaml
oc apply -f install/serving-cert-signer/install.yaml
```

The clusterrole and binding need to move into the operator, but I think that can be a post step.

### CODE REVIEW

**Comment by deads2k** [comment]
/assign @mfojtik 

this is ready for review while I sort out the upstream

File: pkg/controller/servingcert/starter.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		// TODO this needs to be configurable
+		"cluster.local",
+		2*time.Minute,
+	)
+	servingCertUpdateController := NewServiceServingCertUpdateController(
+		kubeInformers.Core().V1().Services(),
+		kubeInformers.Core().V1().Secrets(),
+		kubeClient.CoreV1(),
+		ca,
+		// TODO this needs to be configurable
```
Comment (mfojtik) [inline pkg/controller/servingcert/starter.go]: via Options ?

File: pkg/cmd/servingcertsigner/cmd.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+func NewController() *cobra.Command {
+	o := &ControllerCommandOptions{
+		basicFlags: controllercmd.NewControllerFlags(),
+	}
+
+	cmd := &cobra.Command{
+		Use:   "serving-cert-signer",
+		Short: "Start the Service Serving Cert Signer",
```
Comment (mfojtik) [inline pkg/cmd/servingcertsigner/cmd.go]: nit: "Start the Service Serving Cert Signer controller" ?

**Comment by mfojtik** [comment]
one nit, /lgtm after TODOs are addressed ;-)

File: pkg/controller/servingcert/starter.go (hunk 1 above)
Comment (deads2k) [reply pkg/controller/servingcert/starter.go]: > via Options ?

Yeah. Wasn't there before either, but kube allows it.

**Comment by deads2k** [comment]
nits fixed, deps addressed.  plan to merge on green and keep working from here.

**Pull request merged**